### PR TITLE
Update design from user badges section

### PR DESCRIPTION
**Pull request by arathjz**
**What:**
Update design from user badges section

**Why:**
To address: https://app.asana.com/0/1159164196409129/1186341795801831/f subtask 8

**How:**
- By overwriting style rules to match with new design

Media:
image
image

### CODE REVIEW

File: scss/templates/user.scss [hunk 1]
```diff
@@ -283,3 +283,67 @@
     margin-right: 0;
   }
 }
+
+// User Badges
+.user-content.user-badges-list {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(18.75rem, 1fr));
+  grid-gap: 1.6875rem;
```
Comment (duranmla) [inline scss/templates/user.scss:291]: Why this is not `$grid-gutter-width` ? Be careful between pixel perfect and reproduce the same mistake that the designer potentially can do when creating the idea

Comment (arathjz) [reply scss/templates/user.scss:291]: Fixed @duranmla

File: scss/discourse/button.scss [hunk 2]
```diff
@@ -63,3 +63,25 @@
     cursor: not-allowed;
   }
 }
+
+@mixin user-profile-btn {
```
Comment (orlando) [inline scss/discourse/button.scss:67]: 💯

**Comment by duranmla** [comment]
Shouldn't the badges cards behaves as cards? I mean have the white background and the shadow? @arathjz @orlando

**Pull request merged**